bin/oca_run_tests

```diff
@@ -8,6 +8,11 @@ set -ex
 
 oca_wait_for_postgres
 
+if [ -n "${SERVER_WIDE_MODULES}"]; then
+    WIDE_MODULES=${SERVER_WIDE_MODULES}
```

Comment:
I don't know if we should enhance this to ensure web and base are in the list and concatenate them or to make that in copier